pyne/CMakeLists.txt

```diff
@@ -6,14 +6,27 @@ set(SRC_FILES pyne.cpp)
 # Public headers
 set(PUB_HEADERS pyne.h)
 
-# Linked libraries
-set(LINK_LIBS        ${DAG_LINK_LIBS})
-set(LINK_LIBS_STATIC ${DAG_LINK_LIBS})
+# Optimization flags: always use -O0
+set(CMAKE_CXX_FLAGS_RELEASE "-O0")
```

Comment:
Is this for compile speed? Maybe if build type is Debug we should do O0, maybe we should really optimize it otheriwse? Don't know really, most of the pyne calls arent too deep/slow